src/app/static/src/app/components/modelOutput/ModelOutput.vue

```diff
@@ -99,6 +99,7 @@
                     :selections="comparisonPlotSelections"
                     :formatFunction="formatBarchartValue"
                     :showRangesInTooltips="true"
+                    :no-data-message="noChartData"
```

Comment:
I don't think this is the right place for the message... again see usage https://github.com/mrc-ide/hint/blob/2443d5dccb18440fbdb9dd153b21a28f8c4de91c/src/app/static/src/app/components/genericChart/GenericChart.vue#L34. Should be added in `vue-charts`. If there's no data, vue-charts should display a placeholder with noCharData message. This means we won't have to add the same message everywhere the chart is reused.

Reply:
I disagree because you are assuming that vue-charts is only going to be used within hint. What if it is used in another application and they don't want a no data message, and if they do, why would we assume what sort of text the message should contain? Besides, this approach was agreed when the vue-charts changes were approved

Reply:
Agreed @Nicolas-Dolan - getting the client to specify a message is an approach we already take elsewhere. Also, there's the language issue - we can't expect vue-charts be able to translate the no data message appropriately.